logstash-filter-aggregate.gemspec

```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |s|
   s.description = "This gem is a logstash plugin required to be installed on top of the Logstash core pipeline using $LS_HOME/bin/plugin install gemname. This gem is not a stand-alone program"
   s.authors = ["Elastic", "Fabien Baligand"]
   s.email = 'info@elastic.co'
-  s.homepage = "http://www.elastic.co/guide/en/logstash/current/index.html"
+  s.homepage = "https://github.com/logstash-plugins/logstash-filter-aggregate"
```

Comment:
Well, after giving a bit of thought I think it makes sense to use this. We also open an issue to decide what to do with this, as you noticed, is not consistent. ref: https://github.com/elastic/logstash/issues/3566